### PR TITLE
Downgrade StackExchange.Redis to 1.2.4

### PR DESCRIPTION
**Pull request by jeong760**

### CODE REVIEW

File: src/CoiniumServ/packages.config [hunk 1]
```diff
@@ -11,7 +11,7 @@
   <package id="JsonConfig" version="1.0.0" targetFramework="net47" />
   <package id="JsonFx" version="2.0.1209.2802" targetFramework="net47" />
   <package id="Microsoft.AspNet.Razor" version="3.2.4" targetFramework="net47" />
-  <package id="MySql.Data" version="6.8.8" targetFramework="net47" />
```
Comment (bonesoul) [inline src/CoiniumServ/packages.config:14]: this change may cause bugs actually, as we are strictly staying on mysql.data 6.8.8

can you confirm that everything is okay with 6.9.11?

**Review by bonesoul — CHANGES_REQUESTED** [review]
@jeong760 please don't upgrade mysqldata and let it stay with 6.8.8.

reason; https://github.com/fluentmigrator/fluentmigrator/issues/821

6.9.10 seems good enough.

File: src/CoiniumServ/packages.config (hunk 1 above)
Comment (bonesoul) [reply src/CoiniumServ/packages.config:14]: we should use 6.9.10 or 6.8.8

**Comment by jeong760** [comment]
@Bonesoul,

Ok, I will made a change this to MySql 6.9.10.

Regards,
John Ahn
-----Original Message-----

**Comment by SmarterHomes** [comment]
For the sake of traceability, can the PR be split into fixes and additions?

**Comment by bonesoul** [comment]
I count with that @HashUnlimited

**Pull request closed without merging**